views/play.erb

```diff
@@ -0,0 +1,7 @@
+<h1>Rock, Paper, Scissors</h1>
+Player: <%= @player_name %>
+<form action="/weapon" method="post">
+  Choose your weapon:
+  <input type="text" name="choice" placeholder="rock, paper or scissors">
+  <input type="submit" name="Submit">
+</form>
```

Comment:
Very clear to see what is happening, however there is room for error if the user types something unexpected